terrain_navigation_ros/config/ap_config.yaml

```diff
@@ -0,0 +1,273 @@
+
+# 3dr_radio
+/mavros/**/tdr_radio:
+  ros__parameters:
+    low_rssi: 40  # raw rssi lower level for diagnostics
+
+# actuator_control
+# None
+
+# command
+/mavros/**/cmd:
+  ros__parameters:
+    use_comp_id_system_control: false # quirk for some old FCUs
+
+# dummy
+# None
+
+# ftp
+# None
+
+# global_position
+/mavros/**/global_position:
+  ros__parameters:
+    frame_id: "map"             # origin frame
+    child_frame_id: "base_link" # body-fixed frame
+    rot_covariance: 99999.0   # covariance for attitude?
+    gps_uere: 1.0             # User Equivalent Range Error (UERE) of GPS sensor (m)
+    use_relative_alt: true    # use relative altitude for local coordinates
+    tf.send: false               # send TF?
+    tf.frame_id: "map"  # TF frame_id
+    tf.global_frame_id: "earth"  # TF earth frame_id
+    tf.child_frame_id: "base_link" # TF child_frame_id
+
+# imu_pub
+/mavros/**/imu:
+  ros__parameters:
+    frame_id: "base_link"
+    # need find actual values
+    linear_acceleration_stdev: 0.0003
+    angular_velocity_stdev: 0.0003490659 # 0.02 degrees
+    orientation_stdev: 1.0
+    magnetic_stdev: 0.0
+
+# local_position
+/mavros/**/local_position:
+  ros__parameters:
+    frame_id: "map"
+    tf.send: false
+    tf.frame_id: "map"
+    tf.child_frame_id: "base_link"
+    tf.send_fcu: false
+
+# param
+# None, used for FCU params
+
+# rc_io
+# None
+
+# setpoint_accel
+/mavros/**/setpoint_accel:
+  ros__parameters:
+    send_force: false
+
+# setpoint_attitude
+/mavros/**/setpoint_attitude:
+  ros__parameters:
+    reverse_thrust: false     # allow reversed thrust
+    use_quaternion: false     # enable PoseStamped topic subscriber
+    tf.listen: false           # enable tf listener (disable topic subscribers)
+    tf.frame_id: "map"
+    tf.child_frame_id: "target_attitude"
+    tf.rate_limit: 50.0
+
+# setpoint_raw
+/mavros/**/setpoint_raw:
+  ros__parameters:
+    thrust_scaling: 1.0       # specify thrust scaling (normalized, 0 to 1) for thrust (like PX4)
+
+# setpoint_position
+/mavros/**/setpoint_position:
+  ros__parameters:
+    tf.listen: false           # enable tf listener (disable topic subscribers)
+    tf.frame_id: "map"
+    tf.child_frame_id: "target_position"
+    tf.rate_limit: 50.0
+    mav_frame: LOCAL_NED
+  
+# guided_target
+/mavros/**/guided_target:
+  ros__parameters:
+    tf.listen: false           # enable tf listener (disable topic subscribers)
+    tf.frame_id: "map"
+    tf.child_frame_id: "target_position"
+    tf.rate_limit: 50.0
+
+# setpoint_velocity
+/mavros/**/setpoint_velocity:
+  ros__parameters:
+    mav_frame: LOCAL_NED
+
+# vfr_hud
+# None
+
+# waypoint
+/mavros/**/mission:
+  ros__parameters:
+    pull_after_gcs: true    # update mission if gcs updates
+    use_mission_item_int: true # use the MISSION_ITEM_INT message instead of MISSION_ITEM
+                              # for uploading waypoints to FCU
+  
+
+# --- mavros extras plugins (same order) ---
+
+# /mavros:
```

Comment:
Does this need to be included, or is it left out since mavros is launched with defaults?

Reply:
I may be able to use the default mavros config for ArduPilot but seem to recall that it had a few issues. Maybe we should fix and upstream. The px4 config is directly from mavros.